e2e/hook/config.go

```diff
@@ -0,0 +1,25 @@
+package hook
```

Comment:
maybe we can move this in an internal package (like `hook/internal/config`)? WDYT?

Reply:
Done in 9a83a18.